lib/src/common/repos_service.dart

```diff
@@ -513,28 +515,74 @@ class RepositoriesService extends Service {
         .objects("GET", "/repos/${slug.fullName}/releases", Release.fromJson);
   }
 
-  /// Fetches a single release.
+  /// Fetches a single release by the release ID.
   ///
   /// API docs: https://developer.github.com/v3/repos/releases/#get-a-single-release
-  Future<Release> getRelease(RepositorySlug slug, int id) =>
+  Future<Release> getReleaseById(RepositorySlug slug, int id) =>
       _github.getJSON("/repos/${slug.fullName}/releases/$id",
           convert: Release.fromJson);
 
-  /// Creates a Release based on the specified [release].
+  /// Fetches a single release by the release tag name.
+  ///
+  /// API docs: https://developer.github.com/v3/repos/releases/#get-a-release-by-tag-name
+  Future<Release> getReleaseByTagName(RepositorySlug slug, String tagName) =>
```

Comment:
@robrbecker this is the second method now. We can do the deprecation in this release version